src/Assets/appStory.js

```diff
@@ -0,0 +1,6 @@
+const appStory = {
+  title: "The Story Behind 'Mi Voz, Mi Voto'",
+  text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus dictum dapibus enim at sollicitudin. Proin ac mattis risus. Maecenas iaculis eros vitae enim semper, at egestas massa mattis. Maecenas mattis pharetra magna, viverra sollicitudin tellus pharetra quis. Vestibulum tincidunt congue arcu in laoreet. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc ac malesuada sapien. Aliquam consectetur felis semper velit ullamcorper, sit amet porttitor sem semper. Duis vehicula et nisi vel tempus. Suspendisse nunc diam, porta vulputate ultrices quis, consequat eu leo. Proin nisi quam, semper nec arcu in, semper interdum magna. Cras aliquet fermentum lacus vel posuere.'
+};
+
+export default appStory;
```

Comment:
What do you think about removing this file since we already have a header with the name of the app and our individual stories in regards to why we decided to create this app?